homeassistant/components/smarttub/binary_sensor.py

```diff
@@ -90,3 +97,52 @@ def extra_state_attributes(self):
     def device_class(self) -> str:
         """Return the device class for this entity."""
         return DEVICE_CLASS_PROBLEM
+
+
+class SmartTubError(SmartTubSensorBase):
+    """Indicates whether an error code is present.
+
+    There may be 0 or more errors. If there are >0, we show the first one.
```

Comment:
If there is a better way to represent an arbitrary number of states within the entity model, I'm all ears